packages/components/src/entities/SampleTypeAppContext.tsx

```diff
@@ -15,7 +15,7 @@ import {
 } from '../internal/components/samples/models';
 import { DetailRenderer } from '../internal/components/forms/detail/DetailDisplay';
 
-import { ReferencingNotebooks, SampleStorageMenu, SampleStorageLocation, SampleGridButton } from './models';
+import { ReferencingNotebooks, SampleGridButton, SampleStorageLocation, SampleStorageMenu } from './models';
```

Comment:
I believe `assayProviderType` should be removed from the `SampleTypeAppContext` below, right? Anything else that was moved and should be removed?

Reply:
Ah, no. Gotta add it back to avoid reaching across subpackages